src/main/java/com/scottlogic/competition_engine/application/TestController.java

```diff
@@ -0,0 +1,18 @@
+package com.scottlogic.competition_engine.application;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TestController {
```

Comment:
I understand we are just creating a hello endpoint, but my understanding of this initial deliverable was to create something that can echo the contents of the received token back. I.e. the genesis of what will be our UserService, so it would make sense for this to be the UserService (even if it return a fixed string at present).

As per a previous comment on a different PR, please dont use snake_case for package names, and if this is the userService then a user package would be more appropriate

There are no unit or integration tests

Reply:
Hi James, thank you for the comment! 

- I will write unit / integration tests. 
- I think there has been some miscommunication of what this entails - there is another card for the genesis of user service so I have been working on it as a separate feature on a different branch: https://trello.com/c/70VcVOE2/10-5-back-end-to-have-user-service
Since there is no real use for the hello world endpoint, my suggestion is to close this branch and I'll create another pull request that contains my user service progress with fixed string & tests.